ansible_runner/interface.py

```diff
@@ -897,3 +897,155 @@ def get_ansible_config(action, config_file=None, only_changed=None, **kwargs):
     response = r.stdout.read()
     error = r.stderr.read()
     return response, error
+
+
+def get_role_list(collection=None, playbook_dir=None, **kwargs):
+    '''
+    Run an ``ansible-doc`` command to get list of installed collection roles.
+
+    Only roles that have an argument specification defined are returned.
+
+    .. note:: Version added: 2.2
+
+    :param str collection: A fully qualified collection name used to filter the results.
+    :param str playbook_dir: This parameter is used to sets the relative path to handle playbook adjacent installed roles.
+
+    :param str runner_mode: The applicable values are ``pexpect`` and ``subprocess``. Default is set to ``subprocess``.
+    :param str host_cwd: The host current working directory to be mounted within the container (if enabled) and will be
+                     the work directory within container.
+    :param dict envvars: Environment variables to be used when running Ansible. Environment variables will also be
+                    read from ``env/envvars`` in ``private_data_dir``
+    :param dict passwords: A dictionary containing password prompt patterns and response values used when processing output from
+                      Ansible. Passwords will also be read from ``env/passwords`` in ``private_data_dir``.
+    :param dict settings: A dictionary containing settings values for the ``ansible-runner`` runtime environment. These will also
+                     be read from ``env/settings`` in ``private_data_dir``.
+    :param str ssh_key: The ssh private key passed to ``ssh-agent`` as part of the ansible-playbook run.
+    :param bool quiet: Disable all output
+    :param bool json_mode: Store event data in place of stdout on the console and in the stdout file
+    :param str artifact_dir: The path to the directory where artifacts should live, this defaults to 'artifacts' under the private data dir
+    :param str project_dir: The path to the playbook content, this defaults to 'project' within the private data dir
+    :param int rotate_artifacts: Keep at most n artifact directories, disable with a value of 0 which is the default
+    :param int timeout: The timeout value in seconds that will be passed to either ``pexpect`` of ``subprocess`` invocation
+        (based on ``runner_mode`` selected) while executing command. It the timeout is triggered it will force cancel the execution.
+    :param bool process_isolation: Enable process isolation, using a container engine (e.g. podman).
```

Comment:
[Avoid using latin phrases](https://docs.ansible.com/ansible/latest/dev_guide/style_guide/index.html#avoid-using-latin-phrases).
```suggestion
    :param bool process_isolation: Enable process isolation using a container engine such as podman.
```

Reply:
Done